sktime/forecasting/pytorchforecasting.py

```diff
@@ -17,20 +17,95 @@ class PytorchForecastingTFT(_PytorchForecastingAdapter):
 
     Parameters
     ----------
-    model_params :  Dict[str, Any] (default=None)
+    model_params :  dict[str, Any] (default=None)
         parameters to be passed to initialize the pytorch-forecasting TFT model [1]_
         for example: {"lstm_layers": 3, "hidden_continuous_size": 10}
-    dataset_params : Dict[str, Any] (default=None)
+    dataset_params : dict[str, Any] (default=None)
         parameters to initialize `TimeSeriesDataSet` [2]_ from `pandas.DataFrame`
         max_prediction_length will be overwrite according to fh
         time_idx, target, group_ids, time_varying_known_reals, time_varying_unknown_reals
         will be infered from data, so you do not have to pass them
-    train_to_dataloader_params : Dict[str, Any] (default=None)
+    train_to_dataloader_params : dict[str, Any] (default=None)
         parameters to be passed for `TimeSeriesDataSet.to_dataloader()`
         by default {"train": True}
-    validation_to_dataloader_params : Dict[str, Any] (default=None)
+    validation_to_dataloader_params : dict[str, Any] (default=None)
         parameters to be passed for `TimeSeriesDataSet.to_dataloader()`
         by default {"train": False}
+    model_path: string (default=None)
+        try to load a existing model without fitting. Calling the fit function is
+        still needed, but no real fitting will be performed.
+    random_log_path: bool (default=False)
+        use random root directory for logging. This parameter is for CI test in
+        Github action, not designed for end users.
+
+    Examples
+    --------
+    >>> # import packages
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> from sktime.forecasting.pytorchforecasting import PytorchForecastingTFT
+    >>> from sktime.utils._testing.hierarchical import _make_hierarchical
+    >>> # generate random data
+    >>> data = _make_hierarchical(
+    ...     hierarchy_levels=(5, 200), max_timepoints=50, min_timepoints=50, n_columns=3
+    ... )
+    >>> # define forecast horizon
+    >>> max_prediction_length = 5
+    >>> fh = ForecastingHorizon(range(1, max_prediction_length + 1), is_relative=True)
+    >>> # split X, y data for train and test
+    >>> l1 = data.index.get_level_values(1).map(lambda x: int(x[3:]))
```

Comment:
Can you check if such a split is possible to implement with the InstanceSplitter? This would make the docs much easier to read. If not possible, I think it would be cool to add a detailed comment, since this is visible on the documentation page. Thus, it should be easy to understand.

Reply:
This would apply to all PyTorch forecaster I assume

Reply:
replaced with `sklearn.model_selection.train_test_split`